app/modules/ads/setup.js

```diff
@@ -144,6 +145,13 @@ function setupAdContext(adsContext, instantGlobals, isOptedIn = false) {
     adProductsUtils.mapSamplingResults(instantGlobals.wgAdDriverLABradorDfpKeyvals),
   );
 
+  // Track Labrador values to DW
+  const labradorPropValue = adProductsUtils.getSamplingResults().join(';');
+
+  if (PageTracker.isEnabled() && labradorPropValue) {
+    PageTracker.trackProp('labrador', labradorPropValue);
```

Comment:
Same here - mixed fillings about tracking in context. What about launching it in the module.js (somewhere next to adengine init)?

Reply:
Done.